### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by mattatcha** [mattatcha/terraform-provider-convox#5]

**Comment by nzoschke** [comment]
Still looking for help with the documentation? I'd be happy to learn more about the provider and contribute docs.

**Comment by mattatcha** [comment]
@nzoschke Yes I am! The provider is currently fairly simple so it shouldn't take much work to get docs started.

**Comment by brendanjerwin** [comment]
See: #7
